### PR TITLE
new file:   packages/acp4/acp4.1.1.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by shonfeder** [comment]
Great, tests are building now. But it looks like the build config or deps may need some correction:

```
#=== ERROR while compiling acp4.1.1.0 =========================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/UnixJunkie/ACP4/archive/refs/tags/v1.1.0.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/acp4.1.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p acp4 -j 39
# exit-code            1
# env-file             ~/.opam/log/acp4-7-1d019b.env
# output-file          ~/.opam/log/acp4-7-1d019b.out
### output ###
# File "src/dune", line 13, characters 85-91:
# 13 |   (libraries batteries dolog minicli parany vector3 line_oriented cpm bst ocamlgraph molenc))
#                                                                                           ^^^^^^
# Error: Library "molenc" not found.
``` 

https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/9f55a773dd6042d75caf393fc280f851660717cf/variant/compilers,4.08,acp4.1.1.0

**Comment by UnixJunkie** [comment]
I added the missing dependency; let's see what happens next.

File: packages/acp4/acp4.1.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "batteries" {>= "3.3.0"}
+  "bst" {>= "4.0.0"}
+  "dolog" {> "4.0.0"}
+  "dune" {>= "2.9"}
+  "line_oriented" {>= "1.1.0"}
+  "minicli" {>= "5.0.0"}
+  "ocaml" {>= "4.08"}
+  "parany" {>= "11.0.0"}
+  "vector3"
+  "molenc"
```
Comment (mseri) [inline packages/acp4/acp4.1.1.0/opam:23]:
```suggestion
  "molenc" {>= "16.23.0"}
```
Uses Molenc.Gram

**Comment by shonfeder** [comment]
CI is all good here, except for windows, failing with

```
   #=== ERROR while compiling conf-rdkit.1 =======================================#
  "./test": command not found.
  
  
  <><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
  +- The following actions failed
  | - fetch cmdliner      1.3.0
  | - fetch cpm           12.2.0
  | - fetch cpu           2.0.0
  | - build conf-python-3 9.0.0
  | - build conf-rdkit    1
  +- 
  - No changes have been performed
  
  <><> conf-rdkit.1 troubleshooting <><><><><><><><><><><><><><><><><><><><><><><>
  => This package requires rdkit to be installed. You might want to try:
         'pip3 install rdkit; opam install --fake conf-rdkit'
  # To update the current shell environment, run: (& opam env) -split '\r?\n' | ForEach-Object { Invoke-Expression $_ }
```

This is for a conf- dependency, and there is a clear install failure message. This looks good to go!

Thanks you for publishing the update!

**Pull request merged**